keras/layers/core.py

```diff
@@ -899,3 +903,177 @@ def get_config(self):
                   "input_dim": self.input_dim}
         base_config = super(MaxoutDense, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class Lambda(Layer):
+
+    """Lambda layer for evaluating arbitrary function
+
+    Input shape
+    -----------
+    output_shape of previous layer
+
+    Output shape
+    ------------
+    Specified by output_shape argument
+
+    Arguments
+    ---------
+    function - The function to be evaluated. Takes one argument : ouput of previous layer
```

Comment:
Typo: output. It should be clarified that it needs to be a valid Theano expression.